crates/meroctl/src/cli.rs

```diff
@@ -60,7 +60,7 @@ pub struct RootArgs {
 
     /// Name of node
     #[arg(short, long, value_name = "NAME")]
-    pub node_name: Utf8PathBuf,
+    pub node_name: String,
```

Comment:
are there any arguments in favor of this remaining a path?

Reply:
nope